### ISSUE TITLE
Easy networks validation splitting keyword

### ISSUE THREAD

**Issue by blengerich** [cnellington/Contextualized#158]

**Comment by cnellington** [comment]
Reproducable with for v0.2.4
```
import numpy as np
from contextualized.easy import ContextualizedCorrelationNetworks
C_train = np.random.normal(0, 1, (100, 20))
X_train = np.random.normal(0, 1, (100, 20))
model = ContextualizedCorrelationNetworks(**model_kwargs)
model.fit(C_train, X_train, val_split=.5)
```
Pre-training kwarg validation gives "Received unknown keyword argument val_split, probably ignoring."

**Comment by blengerich** [comment]
Fix in #161
